source/patterns/02-organisms/global/site-header.twig

```diff
@@ -0,0 +1,33 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['site-header']) }) -%}
+
+<header {{- html.attributes(attributes) }}>
+
+  {%- block body -%}
+
+    {%- set logo = logo|default({})|merge({html: html}) -%}
+
+    {%- if logo.image is not defined or logo.image.src is not defined -%}
+      {%- set logo = logo|merge({image: {src: '../../images/logo.svg'}}) -%}
```

Comment:
This allows a default logo is no `logo.image.src` is provided.

Reply:
We need to compute the path to the image; suggest using the `AssetsExtension` in `symfony/twig-bridge`.

Reply:
Though could always embed the SVG... Probably good enough for now, but won't always be appropriate.

Reply:
I've opted for an approach that embeds the svg.